clients/web/src/lib/components/atoms/ProgressBar/ProgressBar.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+    // Obviously, this needs to be its own parameter for the 'filled-ness' of
+    // the bar
+    export let progress = 45;
+
+    // Props to customize which bar you want
+    export let size = "default";
+    export let label = false;
+    export let labelOutside = true;
+    export let progressLabel = "Default Progress Bar Text";
+
+    // Styling choices
+    export let primaryBarColor = "bg-primary-900";
+    export let primaryTextClass = "text-primary-200";
+</script>
+
+{#if label}
+    {#if labelOutside}
+        <div class="flex justify-between mb-1">
+            <span
+                class="text-base font-medium {primaryTextClass}
+            dark:{primaryTextClass}">{progressLabel}</span
```

Comment:
```suggestion
                class="text-base font-medium {primaryTextClass}">{progressLabel}</span
```

Can get rid of `dark:{primaryTextClass}`; that's only needed if the values are going to change

Reply:
This is wrong, per my testing. Not having `dark:class` in the class of the element causes the styling to not be used in our histoire, at least for me locally.

Reply:
Try restarting histoire after making the change; this is likely a bug on their part that has to do with Tailwind running a purge on unused classes

Reply:
removed in latest commit, I got histoire to behave